packages/contracts-bedrock/src/deploy-config.ts

```diff
@@ -136,19 +136,49 @@ interface RequiredDeployConfig {
   proxyAdminOwner: string
 
   /**
-   * L1 address which receives the base fee for the L2 network.
+   * L1 or L2 address which receives the base fee for the L2 network.
```

Comment:
Local or remote

Reply:
@tynes could you clarify what you're suggesting? Do you mean changing these comments to be: `Local or remote address which receives the base fee for the L2 network.`?

Reply:
Yes exactly, I think using local or remote is better language because it generalizes to L3+. Its def not consistent throughout the codebase

Reply:
If I were to read "Local or remote address", personally I think I'd be very confused without context. I think a clearer and still more flexible expression may be something like
```
L1 or higher (eg L2) address which receives the base fee for the L2 network.
```